### ISSUE TITLE
Increasing dimensionality

### ISSUE THREAD

**Issue by alexandreberkovic** [jalola/improved-wgan-pytorch#27]
Hi, I was taking a look at the model and saw several hard-coded 64s but those both represent batch_size and model dimensionality, which ones should be changed in order to increase dimensionality of the GAN ?

**Comment by jalola** [comment]
Hi, You can try `python train.py --help` to see options it supports, there is an option `dim` which is 64 by default. I remember I tried 128, it gives me Out Of Memory for me and when I tried 96, it has some wrong size error. In case it has error because of the size, you need to modify the `ResidualBlock` in both `GoodGenerator` and `GoodDiscriminator`

**Comment by alexandreberkovic** [comment]
That makes sense, would you add a ResidualBlock layer to both Generator and Discriminator to make it work then ?

**Comment by jalola** [comment]
This is already supported as I said, just use `--dim SIZE` so you can train with bigger dimensionality.
I just updated some requirement to support higher Pytorch version.
